arch/arc/kernel/mcount.S

```diff
@@ -0,0 +1,119 @@
+/* SPDX-License-Identifier: GPL-2.0-only */
+/*
+ * Function tracing support for ARC
+ *
+ * Copyright (C) 2023-24 Synopsys, Inc. (www.synopsys.com)
```

Comment:
Ditto.

Reply:
I guess sometimes it is not that great to think ahead! :p Fixed